<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolsWin</class>
 <widget class="QDialog" name="toolsWin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>175</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>添加工具...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
      <property name="text">
       <string>名称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
      <property name="text">
       <string>命令(win32)：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
      <property name="text">
       <string>命令(linux)：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
      <property name="text">
       <string>快捷键：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="T_name">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="T_win32">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="T_linux">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;文泉驿微米黑&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>166</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>T_name</tabstop>
  <tabstop>T_win32</tabstop>
  <tabstop>T_linux</tabstop>
  <tabstop>keySequenceEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>toolsWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
